mappings/net/minecraft/block/Blocks.mapping

```diff
@@ -7,6 +7,18 @@ CLASS net/minecraft/class_2246 net/minecraft/block/Blocks
 	METHOD method_26110 createShulkerBoxBlock (Lnet/minecraft/class_1767;Lnet/minecraft/class_4970$class_2251;)Lnet/minecraft/class_2480;
 		ARG 0 color
 		ARG 1 settings
+	METHOD method_26113 allow (Lnet/minecraft/class_2680;Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;)Z
+		COMMENT A shortcut to allow in a context predicate, used as {@code Blocks::allow}.
```

Comment:
This comment is a bit confusing... It should describe what the method does. What does it allow?

Reply:
It is passed to settings, like `blockVision(Blocks::allow)`

Reply:
Hmm, should prob rename them to always/never

Reply:
I see, so these just always return `true`/`false`?

Reply:
Yep.

Reply:
Yeah, `always` and `never` would be better. `allow` sounds more like a method that adds something to a list of allowed things, than just returning true.